cli/jhipster.js

```diff
@@ -35,6 +35,10 @@ if (!semver.satisfies(currentNodeVersion, minimumNodeVersion)) {
     process.exit(1);
 }
 
+/* WARNING: due to this code, it's a bit more difficult to test when developing
+ * the upgrade generator, workaround consists in running it this way:
+ * node generator-jhipster/cli/cli.js upgrade
+ */
```

Comment:
Yes but I must test that it addresses this particular use case when developing the generator because unfortunately, yarn does not handle binaries well when using link.
I'll give feedback on this.